src/main/kotlin/com/linchpino/core/controller/AccountController.kt

```diff
@@ -29,4 +37,19 @@ class AccountController(private val accountService: AccountService) {
         val result = accountService.createAccount(createAccountRequest)
         return ResponseEntity.status(HttpStatus.CREATED).body(result)
     }
+
+    @Operation(summary = "Search mentors with available timeslots based on date and interviewTypeId")
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Found mentors with available timeslots for provided interviewTypeId and data"),
+            ApiResponse(responseCode = "400", description = "Invalid query parameter")
+        ]
+    )
+    @GetMapping("/mentors/search", produces = [MediaType.APPLICATION_JSON_VALUE])
+    fun findMentorsByInterviewTypeAndDate(
+        @RequestParam(value = "interviewTypeId", required = true) interviewTypeId:Long,
+        @RequestParam(value = "date", required = true) date:LocalDate):ResponseEntity<List<MentorWithClosestTimeSlot>>{
```

Comment:
User only selects date to retrieve mentors with available time slots for that specific date. Since time does not matter in this API using ZonedDateTime is not appropriate

Reply:
SelectedDate and stored from and toDate both should be in UTC to possible to compare together. I agree with you that timezone in not affect the selected date. But in future we will add a combo box to select timezone by job seeker.